charts/rekor-tiles/templates/service.yaml

```diff
@@ -5,10 +5,12 @@ metadata:
   namespace: {{ .Values.namespace.name }}
   labels:
     {{- include "rekor-tiles.labels" . | nindent 4 }}
+  annotations:
+    cloud.google.com/neg: '{"exposed_ports": {"80":{"name": "{{ .Values.neg.name }}"}}}'
```

Comment:
`80` needs to match what's set from `.Values.service.ports[0].port`, correct? Do we need to pull the port num from the template?

Do we also need to specify an ordering for the ports, that the first is for the NEG, second for gRPC, third for metrics (basing this on the current ordering)?